### PR TITLE
[Merged by Bors] - chore: move summable lemmas

### PR DESCRIPTION
**Pull request by CBirkbeck**
We move some lemmas out of `Topology/Instances/ENNReal` into `Topology/Algebra/InfiniteSum/Real`. Also use this to address a porting TODO.

This was originally part of #12446 

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by grunweg — APPROVED** [review]
Looks good to me; thanks for splitting these out!

File: Mathlib/Analysis/SpecificLimits/Basic.lean [hunk 1]
```diff
@@ -6,7 +6,7 @@ Authors: Sébastien Gouëzel, Johannes Hölzl, Yury G. Kudryashov, Patrick Masso
 import Mathlib.Algebra.GeomSum
 import Mathlib.Order.Filter.Archimedean
 import Mathlib.Order.Iterate
-import Mathlib.Topology.Instances.ENNReal
+import Mathlib.Topology.Algebra.InfiniteSum.Real
 import Mathlib.Topology.Algebra.Algebra
```
Comment (Ruben-VandeVelde) [inline Mathlib/Analysis/SpecificLimits/Basic.lean:10]: Please sort

**Review by Ruben-VandeVelde — APPROVED** [review]
This seems okay, though I don't know which direction of dependency is best between these two files.

**Comment by adomani** [comment]
The `no_lost_declarations` script reports

---

* 0  added declarations
* 0  removed declarations
* 9  paired declarations

---

:tada:

**Comment by CBirkbeck** [comment]
So is this ready to go?

**Comment by loefflerd** [comment]
LGTM 

maintainer merge

**Pull request closed without merging**